spec/rubocop/cop/rails/action_filter_spec.rb

```diff
@@ -4,9 +4,12 @@
 
 describe RuboCop::Cop::Rails::ActionFilter, :config do
   subject(:cop) { described_class.new(config) }
+  let(:cop_config) { { 'Include' => nil } }
 
   context 'when style is action' do
-    let(:cop_config) { { 'EnforcedStyle' => 'action' } }
+    before do
```

Comment:
I find that `before(:all)` is used so rarely that the explicitness of `before(:each)` isn’t needed. Besides…
```
» ag 'before do' spec | wc -l
     105
» ag 'before\(:each\) do' spec | wc -l
      14
» ag 'before\(:all\) do' spec | wc -l
       0
```